Sources/SwiftBenchmarkTools/ArgParser.swift

```diff
@@ -1,244 +0,0 @@
-//===----------------------------------------------------------------------===//
```

Comment:
Is the whole `SwiftBenchmarkTools` module getting deleted but GH got confused and not showing that?

Reply:
Yes, the entire module is removed -- if we want to do benchmarks again I'd like to revisit using the new swiftpm plugin based benchmarking tools :)